# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../ipmiutil/no-gzip-manpages.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./doc/Makefile.orig	2007-04-06 22:50:32.000000000 +0200
+++ ./doc/Makefile	2007-04-06 22:50:48.000000000 +0200
@@ -46,5 +46,4 @@
 	${INSTALL_SCRIPT} checksel     ${datato}
 	${INSTALL_SCRIPT} wdt.sh       ${datato}
 	${INSTALL_SCRIPT} ipmi_if.sh   ${datato}
-	cd ${DESTDIR}${mandir}/man8; gzip -f *.8
 	
